src/installer/pkg/projects/netcoreapp/src/netcoreapp.depproj

```diff
@@ -70,6 +70,42 @@
     </ItemGroup>
   </Target>
 
+  <!-- Platform manifest needs to include all System*Native items from 3.1 release. This target adds them explicitly,
+      to work around the issue introduced with a fix for https://github.com/dotnet/runtime/issues/33118,
+      Android libraries need to conform to a specific naming convention which necessitated renames of these libraries.
+      For more details of the issue and fix see https://github.com/dotnet/runtime/issues/33450 -->
+
+  <Target Name="AddSharedFrameworkNativeFilesWithoutLibPrefixToPlatformManifest"
```

Comment:
Why are you using a target for this?  Can't this all be declared statically?

Maybe something like this:
```
<ItemGroup>
  <_pastShimFilesNames Include="System.Native;System.Net.Http.Native;System.Security.Cryptography.Native.OpenSsl;System.Gobalization.Native" />
  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).so')" />
  <_pastShimFilesNames Include="System.Security.Cryptography.Native.Apple" />
  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).a')" />
  <_pastShimFiles Include="@(_pastShimFilesNames->'%(Identity).dylib')" />
  <SharedFrameworkRuntimeFiles Include="@(_pastShimFiles)" TargetPath="runtimes/" />
</ItemGroup>
```

Reply:
A target would ensure these entries were added to `SharedFrameworkRuntimeFiles` before it's consumed in `GenerateFileVersionProps`

Reply:
The project evaluation phase completes before any task is run. (I can't find a page that specifically mentions this about evaluation, but https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-items?view=vs-2019#create-items-during-execution comes very close.) I agree putting them down as static items is better if it works, here.

(Generating the names based on patterns is also nice to see the intent rather than a simple list.)

Reply:
Cool, thanks! I will update the PR to incorporate this suggested change.